services/slam/builtin/orbslam_int_test.go

```diff
@@ -62,7 +62,7 @@ func testPositionAndMap(t *testing.T, svc slam.Service) {
 	// Typical values are around (0.001, -0.001, 0.007)
 	t.Logf("Position point: (%v, %v, %v)",
 		position.Pose().Point().X, position.Pose().Point().Y, position.Pose().Point().Z)
-	// Typical values are around (-0.73, 0.34, 0.58), theta=2.6
+	// Typical values are around (0.001, 0.977, 0.215), theta=0.002
```

Comment:
Same here, don't know what those values mean. Might be good to get a sanity check on this with someone who is familiar with Viam's orientation vector.

Reply:
will add peter as a reviewer